test/e2e/support/utils/convert-block.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Convert editor's block type.
```

Comment:
Can we use the form of 3rd person for consistency?

`Converts ...`

Reply:
Sure, thx.